funcbench/env.go

```diff
@@ -63,11 +64,17 @@ func newLocalEnv(e environment) (Environment, error) {
 	return &Local{environment: e, repo: r}, nil
 }
 
-func (l *Local) PostErr(context.Context, string) error { return nil } // Noop. We will see error anyway.
+func (l *Local) PostErr(string) error { return nil } // Noop. We will see error anyway.
 
-func (l *Local) PostResults(ctx context.Context, cmps []BenchCmp, extraInfo ...string) error {
+func (l *Local) PostResults(tables []*benchstat.Table, extraInfo ...string) error {
 	fmt.Println("Results:")
-	Render(os.Stdout, cmps, true, true, l.compareTarget)
+	var buf bytes.Buffer
+	benchstat.FormatText(&buf, tables)
+
+	if _, err := os.Stdout.Write(buf.Bytes()); err != nil {
```

Comment:
Could we not just use `fmt.Println` here?

Reply:
Using `os.Stdout` to print a buffer is much straight forword.
`fmt.Println` will call `os.Stdout.Write` in the end. Plus, have to covert from []byte to string, and string back to []byte.

Reply:
If we're annoyed with the error check, it can be saved here, just like normal `fmt.Println` doesn't check the error.